dags/veda_data_pipeline/utils/s3_discovery.py

```diff
@@ -111,12 +111,10 @@ def construct_single_asset_items(discovered_files: List[str]) -> dict:
     for uri in discovered_files:
         # Each file gets its matched asset type and id
         filename = uri.split("/")[-1]
-        filename_without_extensions = Path(filename)
-        while filename_without_extensions.suffix: #for filenames that have multiple extensions i.e. *.tif.cog
-             filename_without_extensions=filename_without_extensions.with_suffix('')
+        filename_without_extension = Path(filename).stem
         prefix = "/".join(uri.split("/")[:-1])
         item = {
-            "item_id": str(filename_without_extensions),
+            "item_id": filename_without_extension,
             "assets": {
                 "default": {
```

Comment:
I hadn't noticed this earlier and it is not part of the change proposed in this PR, but: does this mean that when more specific asset info in the discovery config we are making items with an asset key `"default"` instead of `"cog_default"`?  Recent ingests to dev still produce items with cog_default assets so this is probably not a relevant comment

Reply:
@anayeaye I think this is the where the `default` gets changed to `cog_default`: https://github.com/NASA-IMPACT/veda-data-airflow/blob/62ac7b1ae0fab843694fd1b2ef8d4066d5bf0cac/docker_tasks/build_stac/utils/stac.py#L120